lib/di/init_dependencies.dart

```diff
@@ -2,9 +2,17 @@
 import 'package:data/data.dart';
 import 'package:domain/domain.dart';
 import 'package:presentation/presentation.dart';
+import 'package:star_movie/key_store_loader/key_store_loader.dart';
 
-void initDependencies() {
+const keysPath = 'keys.json';
+
+Future<void> initDependencies() async {
   initPresentationModule();
-  initDomainDependencies();
+  initDomainDependencies(await keys());
   initDataDependencies();
 }
+
+Future<Map<String,dynamic>> keys() async{
+  final keyStoreLoader = KeyStoreLoader(path: keysPath);
+  return keyStoreLoader.load();
+}
```

Comment:
1) can we move this logic from root's di file into data's di
2) whole keys logic should be in data in this case, not in the domain

Reply:
I did it but it seems not very pretty, because I forced to pass omdbApiKey as String to domain layer from data thru GetIt. Is this good decision?

Reply:
why do u need it in domain? it seems to be the data logic